_posts/2020-05-08-Rails-6-1-beta1-release.markdown

```diff
@@ -0,0 +1,58 @@
+
+## Strict Loading Associations
+
+In addition to all the database and connection management improvements, [Aaron Patterson](https://github.com/tenderlove) and [Eileen M. Uchitelle](https://github.com/eileencodes) added support for [strict loading associations](https://github.com/rails/rails/pull/37400). With this feature you can ensure that all your associations are loaded eagerly and stop N+1's before they happen. [Kevin Deisz](https://github.com/kddeisz) added additional support to [association declarations](https://github.com/rails/rails/pull/38541).
+
+## Destroy Associations Later
+
+Destroy Associations Later adds `ActiveRecord::Base.destroy_later` which tells your application that you want to run `destroy` in a background job. This can help you avoid timeouts in your application. The [implementation](https://github.com/rails/rails/pull/39149) was a group effort - the PR was started by [George Claghorn](https://github.com/georgeclaghorn) from Basecamp, further support added by [Cory Gwin](https://github.com/gwincr11) of GitHub and finalized by [Rafael França](https://github.com/rafaelfranca) from Shopify.
+
+## Error Objects
+
+Active Model's errors are now objects with an interface that allows your application to more easily handle and interact with errors thrown by models. [The feature](https://github.com/rails/rails/pull/32313) was implemented by [lulalala](https://github.com/lulalala) and includes a query interface, enables more precise testing, and access to error details.
+
+## Active Storage Improvements
+
+Active Storage got a nice update in Rails 6.1! You can now configure attachments for service you want to store them in.The [feature](https://github.com/rails/rails/pull/34935) was implemented by [Dmitry Tsepelev](https://github.com/DmitryTsepelev).
+
+Additionally, Rails 6.1 adds support to Active Storage for [permanent URLs for blobs](https://github.com/rails/rails/pull/36729). Implemented by [Peter Zhu](https://github.com/peterzhu2118) from Shopify, this feature allows configuring your attachments to use a private or public URL and ensures that public URLs will always use a permanent URL.
+
```

Comment:
Any love for https://github.com/rails/rails/pull/34477? I am personally very excited for this feature, and I think others will be too!